### ISSUE TITLE
Can't start librespot

### ISSUE THREAD

**Issue by WyntrHeart** [kraxarn/spotify-qt#248]
### System information
ABI             : x86_64-little_endian-llp64
App version     : v3.11 (83bfc5d)
Build type      : Release
D-Bus support   : Yes
KCrash support  : No
Kernel          : winnt 10.0.19045
Keychain support: Yes
Platform        : windows
Product         : Windows 10 Version 22H2
Qt version      : 6.6.1

### Detailed information
I installed Spotify-Qt via Scoop, and Librespot via the "cargo install" command.
Spotify-Qt will not start librespot. If I set "Start With App" then restart Spotify-Qt it says that librespot is not running. If I click "Start Client" then it says "Please Wait..." forever and never actually starts.
The weirdest part is that the first time I tried it, it actually worked flawlessly, but then after making no changes whatsoever, just closing Spotify-Qt and restarting it later, it no longer worked. I've tried reinstalling completely including wiping config files, rebooting my PC, and using the previous version (3.10) of Spotify-Qt. No luck. Running librespot manually from the command prompt seems to work just fine, although Spotify-Qt does not detect that it's running. The logs menu shows nothing at all, they're empty.

### What happens?
librespot does not start when Spotify-Qt asks it to

### What should happen?
librespot should start when Spotify-Qt asks it to

**Comment by eufcca2** [comment]
I'm running into the same issue with spotify-qt installed via scoop and librespot via cargo-binstall.

### System information

ABI             : x86_64-little_endian-llp64
App version     : v3.11 (83bfc5d)
Build type      : Release
D-Bus support   : Yes
KCrash support  : No
Kernel          : winnt 10.0.19044
Keychain support: Yes
Platform        : windows
Product         : Windows 10 Version 21H2
Qt version      : 6.6.1

### Detailed information

Ditto

### What happens?

Ditto

### What should happen?

Ditto

**Comment by stassss2011** [comment]
I resolved the issue by filling Username field on Configuration tab

**Comment by freddycansic** [comment]
> I resolved the issue by filling Username field on Configuration tab

This fixed it for me

**Comment by martinvonwittich** [comment]
>  Spotify-Qt will not start librespot. If I set "Start With App" then restart Spotify-Qt it says that librespot is not running. If I click "Start Client" then it says "Please Wait..." forever and never actually starts.

I have the same problem. The whole concept is totally confusing to me - why doesn't it simply start the binary? Why do I need to configure a username? Why does it prompt for a password when I enter my username? spotify-qt is running under my username, why does it need my username and password?! Nothing about this seems documented.

**Comment by WyntrHeart** [comment]
Yeah, logging in again via the librespot settings fixed the issue, but this seems both unclear and redundant when spotify-qt is already logged into my account. Why not just automatically use the same credentials to log in for librespot?

**Comment by kraxarn** [comment]
spotify-qt never receives your username or password and can only access the Web API. Actual playback from another client requires additional access that can't be received from just Web API access. Username could probably be guessed from the API though, but password would still be required as that's not something spotify-qt ever receives.

Either way, having to go to a separate tab to enter the username might not be the clearest thing ever, so some improvements can certainly be made. Thanks for bringing up the issue!

**Comment by JamesBlast** [comment]
Hey!  I've installed following the same steps as above, logging fixed the librespot not starting issue  and  it says running  but I can't see any devices matching my arguments... So can't play music locally yet (arguments worked when starting librespot alone so should be ok). Any idea?

**Comment by JamesBlast** [comment]
Ok so it seems that I was wrong,  I get errors because of the arguments which is weird because it wasn't an issue for librespot in CLI. 
So the only argument I kept because it matters is written as follows:
 --device 'Speakers 03 (ASIOVADPRO Driver)'

 I tried w/o ' ' and w/ " (this one worked in cli) for the  device's name
 I get this error:
 
[spotify-qt-20240711.log](https://github.com/user-attachments/files/16181395/spotify-qt-20240711.log)

which is weird because with  : 

--device ?

I get the proper name for the driver in the list, I need to send music to this driver, this is a feature I can't do w/o

**Comment by kraxarn** [comment]
Do you mean "--device-name" or "--device-type"? You seem to mention the device name, while the error seems to be more about the device type.

Also, please open a new issue as this is a different problem, thanks!

**Comment by kraxarn** [comment]
Closing as this is no longer a supported login method
